rfc/text/advanced/ap_50_payload_passthru_mode.md

```diff
@@ -162,11 +166,14 @@ without guessing. The format of the value may depend on the `ppt_scheme` attribu
 | Attribute      | Required? | Value                       |
 |----------------|-----------|-----------------------------|
 | ppt_scheme     | Y         | wamp.eth                    |
-| ppt_serializer | N         | cbor, flatbuffers           |
+| ppt_serializer | N         | cbor, flatbuffers, native*  |
 | ppt_cipher     | N         | xsalsa20poly1305, aes256gcm |
-| ppt_keyid      | N         | *                           |
+| ppt_keyid      | N         | **                          |
+
+*: If `ppt_serializer` is not provided then it is assuming as `native`. So payload will be serialized 
```

Comment:
this is a new meaning for "native": use the same serializer as in _the_ enclosing WAMP session - which doesn't make sense, as there are at least 2 such sessions

"native" means: defined outside of WAMP by the ppt_scheme in use, and the specific scheme "wamp.eth" only allows "cbor" and "flatbuffers" - not "native"

Reply:
> "wamp.eth" only allows "cbor" and "flatbuffers" - not "native"
Why not allow any serializer with binary support? E.g. why msgpack is bad?

Reply:
So i think we should mark ppt_serializer as required for `wamp.eth` scheme, as it allows a few possible values

Reply:
> Why not allow any serializer with binary support? E.g. why msgpack is bad?

cbor and flatbuffers both support binaries. msgpack isn't actually bad .. we could add it. however, we need to be careful, since with e2ee each peer determines the serializer used when sending - and hence the other peer must be prepared to support all serializers. adding msgpack would require support in each client. this is different from regular WAMP - exactly because with regular WAMP the router transparently transcodes different serializers between 2 sessions involved in a WAMP action